### PR TITLE
feat: typography link style

### PR DESCRIPTION
**Pull request by HavardNot**
Added Typography style for link as according to the EDS guidelines. Placed under group "interactive"

### CODE REVIEW

**Review by HaakonSvane — APPROVED** [review]
Good stuff cowboy 🐮

**Review by tormodAase — COMMENTED** [review]
Looks good! Haakon should take a look before merging

**Pull request merged**
